collectd/build.gradle

```diff
@@ -36,15 +43,11 @@ if (!collectDLibPath.exists()) {
 archivesBaseName = 'applicationinsights-collectd'
 
 dependencies {
-    compile project(':core')
+    compile group: 'com.microsoft.azure', name: 'applicationinsights-core', version: '0.9.5'
```

Comment:
Why not using project dependency?
If you have no choice but to use external dependency then please do not use hard-coded fixed version.